### PR TITLE
chore(issue-template): Add link to Anki Forums

### PR DESCRIPTION
**Pull request by david-allison**
This is to reduce the number of upstream issues we receive

A discussion on issue 15810 caused this suggestion

* #15810

## Testing

https://github.com/david-allison/Anki-Android/issues/new/choose

Screenshot 2024-03-08 at 18 08 32

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by BrayanDSO — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Anki Forums
+    url: https://forums.ankiweb.net/
+    about: If your issue also occurs in Anki Desktop, please report it here
```
Comment (BrayanDSO) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: If your issue also occurs in Anki Desktop, please report it there
```

**Review by lukstbit — APPROVED** [review]

**Comment by dae** [comment]
@david-allison @mikehardy currently the AnkiDroid section of the Anki forums says "this is not AnkiDroid's preferred support location". Have you guys considered retiring the (not-very-user-friendly) Google group and moving over to the forums instead? That would make it easier to move discussions to the right section, and reduce user confusion.

**Comment by david-allison** [comment]
@dae AnkiDroid points to Anki Forums as the first element of support in the in-app menu. I'll confirm our Facebook auto-reply also goes there.

We need somewhere that our Play Store developer contact email can go to, which has strong anti-spam measures (messages can be gated before being published). Google groups works excellently for this, it also doesn't require an account before posting, and supports attachments.

**Comment by dae** [comment]
Would you like me to remove the 'not preferred support location' message on the forum category, or keep it?

**Comment by david-allison** [comment]
I'm happy to make it official

**Pull request merged**